lib/extensions/ar_taggable.rb

```diff
@@ -178,7 +178,7 @@ def is_vtagged_with?(tag, options={})
         macro = subject.class.reflect_on_association(relationship.to_sym).macro
       end
       if macro == :has_one || macro == :belongs_to
-        value = subject.instance_eval([relationship, attr].join("."))
+        value = subject.public_send(relationship).public_send(attr)
```

Comment:
We just passed it to `reflect_on_association`.. and use a straight `send` in the else clause.

The latter reasoning seems to apply to `attr` too.